### PR TITLE
Updated CIS references to align with RHEL7 v2.2.0 and RHEL8 v1.0.0 benchmarks

### PR DESCRIPTION
**Pull request by matusmarhefka**
#### Description:

- This PR updates CIS references which were not aligned with the RHEL7 v2.2.0 and RHEL8 v1.0.0 benchmarks.

### CODE REVIEW

**Comment by ggbecker** [comment]
I wonder if some of the older references are actually applicable to different products. Although it looks like many of them were wrongly assigned (probably a copy-paste problem).

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/rule.yml [hunk 1]
```diff
@@ -31,7 +31,8 @@ identifiers:
 
 references:
     stigid@ol7: OL07-00-010270
-    cis: 5.3.3
+    cis@rhel7: 5.3.3
```
Comment (yuumasato) [inline linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/rule.yml:34]: The `rhel7`reference is `5.3.4.
```suggestion
    cis@rhel7: 5.3.4
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_creat/rule.yml [hunk 2]
```diff
@@ -40,7 +40,8 @@ identifiers:
 
 references:
     stigid@ol7: OL07-00-030500
-    cis: 5.2.10
+    cis@rhel7: 4.1.10
+    cis@rhel8: 4.1.9
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_creat/rule.yml:44]:
```suggestion
    cis@rhel7: 4.1.11
    cis@rhel8: 4.1.10
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_ftruncate/rule.yml [hunk 3]
```diff
@@ -43,7 +43,8 @@ identifiers:
 
 references:
     stigid@ol7: OL07-00-030550
-    cis: 5.2.10
+    cis@rhel7: 4.1.10
+    cis@rhel8: 4.1.9
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_ftruncate/rule.yml:47]:
```suggestion
    cis@rhel7: 4.1.11
    cis@rhel8: 4.1.10
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open/rule.yml [hunk 4]
```diff
@@ -43,7 +43,8 @@ identifiers:
 
 references:
     stigid@ol7: OL07-00-030510
-    cis: 5.2.10
+    cis@rhel7: 4.1.10
+    cis@rhel8: 4.1.9
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open/rule.yml:47]:
```suggestion
    cis@rhel7: 4.1.11
    cis@rhel8: 4.1.10
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open_by_handle_at/rule.yml [hunk 5]
```diff
@@ -40,7 +40,8 @@ identifiers:
 
 references:
     stigid@ol7: OL07-00-030530
-    cis: 5.2.10
+    cis@rhel7: 4.1.10
+    cis@rhel8: 4.1.9
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open_by_handle_at/rule.yml:44]:
```suggestion
    cis@rhel7: 4.1.11
    cis@rhel8: 4.1.10
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands/rule.yml [hunk 6]
```diff
@@ -43,7 +43,8 @@ identifiers:
 
 references:
     stigid@ol7: OL07-00-030360
-    cis: 5.2.10
+    cis@rhel7: 4.1.10
+    cis@rhel8: 4.1.9
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands/rule.yml:47]:
```suggestion
    cis@rhel7: 4.1.12
    cis@rhel8: 4.1.13
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlinkat/rule.yml [hunk 7]
```diff
@@ -34,7 +34,8 @@ identifiers:
     cce@rhcos4: CCE-82653-7
 
 references:
-    cis: 5.2.10
+    cis@rhel7: 4.1.11
+    cis@rhel8: 4.1.10
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlinkat/rule.yml:38]:
```suggestion
    cis@rhel7: 4.1.14
    cis@rhel8: 4.1.14
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlink/rule.yml [hunk 8]
```diff
@@ -34,7 +34,8 @@ identifiers:
     cce@rhcos4: CCE-82652-9
 
 references:
-    cis: 5.2.10
+    cis@rhel7: 4.1.11
+    cis@rhel8: 4.1.10
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_unlink/rule.yml:38]:
```suggestion
    cis@rhel7: 4.1.14
    cis@rhel8: 4.1.14
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_truncate/rule.yml [hunk 9]
```diff
@@ -44,7 +44,8 @@ identifiers:
 
 references:
     stigid@ol7: OL07-00-030540
-    cis: 5.2.10
+    cis@rhel7: 4.1.10
+    cis@rhel8: 4.1.9
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_truncate/rule.yml:48]:
```suggestion
    cis@rhel7: 4.1.11
    cis@rhel8: 4.1.10
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_renameat/rule.yml [hunk 10]
```diff
@@ -32,7 +32,8 @@ identifiers:
     cce@rhcos4: CCE-82649-5
 
 references:
-    cis: 5.2.10
+    cis@rhel7: 4.1.11
+    cis@rhel8: 4.1.10
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_renameat/rule.yml:36]:
```suggestion
    cis@rhel7: 4.1.14
    cis@rhel8: 4.1.14
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_rename/rule.yml [hunk 11]
```diff
@@ -32,7 +32,8 @@ identifiers:
     cce@rhcos4: CCE-82648-7
 
 references:
-    cis: 5.2.10
+    cis@rhel7: 4.1.11
+    cis@rhel8: 4.1.10
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_rename/rule.yml:36]:
```suggestion
    cis@rhel7: 4.1.14
    cis@rhel8: 4.1.14
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_openat/rule.yml [hunk 12]
```diff
@@ -43,7 +43,8 @@ identifiers:
 
 references:
     stigid@ol7: OL07-00-030520
-    cis: 5.2.10
+    cis@rhel7: 4.1.10
+    cis@rhel8: 4.1.9
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_openat/rule.yml:47]:
```suggestion
    cis@rhel7: 4.1.11
    cis@rhel8: 4.1.10
```

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open_by_handle_at_rule_order/rule.yml [hunk 13]
```diff
@@ -50,7 +50,8 @@ identifiers:
     cce@rhcos4: CCE-82643-8
 
 references:
-    cis: 5.2.10
+    cis@rhel7: 4.1.11
+    cis@rhel8: 4.1.10
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open_by_handle_at_rule_order/rule.yml:54]: The `*_rule_order` rules are not selected by CIS profiles.

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open_by_handle_at_o_creat/rule.yml [hunk 14]
```diff
@@ -44,7 +44,8 @@ identifiers:
     cce@rhcos4: CCE-82641-2
 
 references:
-    cis: 5.2.10
+    cis@rhel7: 4.1.11
+    cis@rhel8: 4.1.10
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open_by_handle_at_o_creat/rule.yml:48]: The `*_o_creat` rules are not selected by CIS profiles.

File: linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open_by_handle_at_o_trunc_write/rule.yml [hunk 15]
```diff
@@ -43,7 +43,8 @@ identifiers:
     cce@rhcos4: CCE-82642-0
 
 references:
-    cis: 5.2.10
+    cis@rhel7: 4.1.11
+    cis@rhel8: 4.1.10
```
Comment (yuumasato) [inline linux_os/guide/system/auditing/auditd_configure_rules/audit_file_modification/audit_rules_unsuccessful_file_modification_open_by_handle_at_o_trunc_write/rule.yml:47]: The `*_o_trunc_write` rules are not selected by CIS profiles.

**Review by yuumasato — CHANGES_REQUESTED** [review]
A few corrections, please double check them, :)

Some of the rules are not used by the CIS profiles...
Do you think it makes sense to remove the references?

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/rule.yml (hunk 1 above)
Comment (matusmarhefka) [reply linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/rule.yml:34]: Are you sure? The `accounts_password_pam_unix_remember` rule sets `pam_pwhistory` which is under `5.3.3` in RHEL7 benchmark, the `5.3.4` as you suggested sets password encryption to sha512.

**Comment by matusmarhefka** [comment]
> A few corrections, please double check them, :)
> 
> Some of the rules are not used by the CIS profiles...
> Do you think it makes sense to remove the references?

Regarding rules which are not selected by CIS profiles, what do you say if we remove the references from them? I think it will be more consistent than keeping references and we can always add references once rules are selected by the profile.

File: linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/rule.yml (hunk 1 above)
Comment (yuumasato) [reply linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_password_pam_unix_remember/rule.yml:34]: You are correct, :)

**Comment by yuumasato** [comment]
> Regarding rules which are not selected by CIS profiles, what do you say if we remove the references from them?

I say yes. I'm in favor of removing them.

**Comment by yuumasato** [comment]
Tip: If you go to "Files Changed" Tab, GitHub will allow you to batch the suggestion updates.
Yes, the UX could be better, :laughing:

**Review by yuumasato — APPROVED** [review]
Thank you @matusmarhefka

**Pull request merged**